app/services/images/optimizer.rb

```diff
@@ -35,12 +35,26 @@ def self.cloudinary(img_src, **kwargs)
 
     def self.imgproxy(img_src, **kwargs)
       options = DEFAULT_IMGPROXY_OPTIONS.merge(kwargs).reject { |_, v| v.blank? }
-
+      Imgproxy.config.endpoint ||= set_imgproxy_endpoint
       Imgproxy.url_for(img_src, options)
     end
 
     def self.imgproxy_enabled?
       Imgproxy.config.key.present? && Imgproxy.config.salt.present?
     end
+
+    def self.set_imgproxy_endpoint
```

Comment:
**nitpick (non-blocking):** The method doesn't actually set the endpoint, it just returns it.

Reply:
that's a great point. let me update it to `get_imgproxy_endpoint`